# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/builtin/DataViewObject.cpp
# Commit: 410e13fdf4e5
# Full Hash: 410e13fdf4e5927cf6ab312e77f00926a5669508
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 1
# Description:
#   Bug 1720422 - Move typed array unwrap API to templatized version r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119856
# ==============================================================================

diff -r 677e4aa42b4d -r 410e13fdf4e5 js/src/builtin/DataViewObject.cpp
--- a/js/src/builtin/DataViewObject.cpp	Thu Aug 12 20:44:15 2021 +0000
+++ b/js/src/builtin/DataViewObject.cpp	Thu Aug 12 20:44:15 2021 +0000
@@ -966,6 +966,8 @@
         JSCLASS_HAS_CACHED_PROTO(JSProto_DataView),
     &DataViewObjectClassOps, &DataViewObject::classSpec_};
 
+const JSClass* const JS::DataView::ClassPtr = &DataViewObject::class_;
+
 const JSClass DataViewObject::protoClass_ = {
     "DataView.prototype", JSCLASS_HAS_CACHED_PROTO(JSProto_DataView),
     JS_NULL_CLASS_OPS, &DataViewObject::classSpec_};